### ISSUE TITLE
SDK does not install on CentOS 7.5 with Python 2.7.5

### ISSUE THREAD

**Issue by DamaniN** [rubrik-devops/rubrik-sdk-for-python#1]
Trying to install on CentOS 7.5 with Python 2.7.5 but the package does not install. Works on the same system with Python 3.6.5.

Error:

```
[centos@ip-10-10-2-211 rubrik-sdk-for-python]$ more /etc/redhat-release
CentOS Linux release 7.5.1804 (Core)
[centos@ip-10-10-2-211 rubrik-sdk-for-python]$ python --version
Python 2.7.5

[centos@ip-10-10-2-211 rubrik-sdk-for-python]$ sudo python setup.py install
running install
running bdist_egg
running egg_info
writing UNKNOWN.egg-info/PKG-INFO
writing top-level names to UNKNOWN.egg-info/top_level.txt
writing dependency_links to UNKNOWN.egg-info/dependency_links.txt
reading manifest file 'UNKNOWN.egg-info/SOURCES.txt'
writing manifest file 'UNKNOWN.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/rubrik
copying build/lib/rubrik/__init__.py -> build/bdist.linux-x86_64/egg/rubrik
copying build/lib/rubrik/api.py -> build/bdist.linux-x86_64/egg/rubrik
copying build/lib/rubrik/cluster.py -> build/bdist.linux-x86_64/egg/rubrik
copying build/lib/rubrik/rubrik.py -> build/bdist.linux-x86_64/egg/rubrik
byte-compiling build/bdist.linux-x86_64/egg/rubrik/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-x86_64/egg/rubrik/api.py to api.pyc
byte-compiling build/bdist.linux-x86_64/egg/rubrik/cluster.py to cluster.pyc
byte-compiling build/bdist.linux-x86_64/egg/rubrik/rubrik.py to rubrik.pyc
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying UNKNOWN.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying UNKNOWN.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying UNKNOWN.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying UNKNOWN.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating 'dist/UNKNOWN-0.0.0-py2.7.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing UNKNOWN-0.0.0-py2.7.egg
Removing /usr/lib/python2.7/site-packages/UNKNOWN-0.0.0-py2.7.egg
Copying UNKNOWN-0.0.0-py2.7.egg to /usr/lib/python2.7/site-packages
UNKNOWN 0.0.0 is already the active version in easy-install.pth

Installed /usr/lib/python2.7/site-packages/UNKNOWN-0.0.0-py2.7.egg
Processing dependencies for UNKNOWN==0.0.0
Finished processing dependencies for UNKNOWN==0.0.0
[centos@ip-10-10-2-211 rubrik-sdk-for-python]$
```

**Comment by drew-russell** [comment]
```
Collecting rubrik-cdm
  Downloading https://files.pythonhosted.org/packages/2b/c9/e9df757bed8363fd36db451ce546790d6da9a0196aa2952c6d3d01467235/rubrik_cdm-1.0.6.tar.gz
    Complete output from command python setup.py egg_info:
    /usr/lib64/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'long_description_content_type'
      warnings.warn(msg)
    error in rubrik_cdm setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers
```

**Comment by drew-russell** [comment]
Issue has been resolved in the 1.0.7 release.
